package.json

```diff
@@ -24,10 +25,11 @@
         "typescript": "^5.0.2"
     },
     "scripts": {
-        "install": "node-gyp-build \"node scripts/download-CanBridge.mjs\"",
-        "prepublishOnly": "node scripts/download-CanBridge.mjs && tsc && prebuildify --napi",
-        "pretest": "node-gyp-build && tsc",
-        "test": "node --napi-modules test/test_binding.js"
+        "install": "pkg-prebuilds-verify binding-options.cjs || npm run build",
+        "build": "node scripts/download-CanBridge.mjs && cmake-js compile && tsc",
+        "prepublishOnly": "npm run build && pkg-prebuilds-copy --baseDir build/Release --source node_canbridge.node --name node_canbridge --napi_version=8",
```

Comment:
Other platforms besides windows need baseDir set to build, if I imagine correctly